.github/workflows/binary-compatibility-checks.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         # The versions of scala specified here are only used as `+~...`
         # which ignores the PATCH portion of the version id. These Scala versions don't need
         # to be fully defined here then since Akka build handles which patch version will be used.
-        scalaVersion: [ "2.12", "2.13", "3.1" ]
+        scalaVersion: [ "2.12", "2.13", "3.2" ]
```

Comment:
When we change such versions we must also update the branch protection rules since we have the version in the name (which isn't a great practise for this reason). I have changed it, so just fyi.

"Check / Binary Compatibility (3.2)"

Reply:
I think I did that change in exact parallel with you :D